sapi/fpm/fpm/fpm_scoreboard.h

```diff
@@ -52,6 +54,8 @@ struct fpm_scoreboard_s {
 		atomic_t lock;
 		char dummy[16];
 	};
+	atomic_t writer_active;
+	atomic_t reader_count;
```

Comment:
`reader_count` is always accessed when `scoreboard->lock` is acquired, so it doesn't have to be atomic_t. And AFAICS the type name is even a lie because it just means the variable is volatile, not atomic?

Reply:
Yeah you are right. I started with atomic reading thinking that some part might not need lock but it wasn't actually the case. I will change it.

Reply:
I changed it to unsigned.